modules/protobuf/29.0-rc2.bcr.1/MODULE.bazel

```diff
@@ -0,0 +1,181 @@
+# TODO: migrate all dependencies from WORKSPACE to MODULE.bazel
+# https://github.com/protocolbuffers/protobuf/issues/14313
+
+module(
+    name = "protobuf",
+    version = "29.0-rc2.bcr.1",  # Automatically updated on release
```

Comment:
I think also missing a .patch for this version change: https://buildkite.com/bazel/bcr-presubmit/builds/8408#0192d99e-7b45-4c98-ba8a-849a744a0156

Reply:
Fixed in latest commit